users-api/app.js

```diff
@@ -7,12 +7,19 @@ const bodyParser = require('body-parser');
 //create app
 const app = express();
 
+<<<<<<< HEAD
```

Comment:
you need to fix this. This is a conflict. Choose the portions of code that reflect your changes and mine